pkgs/atc0005/send2teams/registry.yaml

```diff
@@ -0,0 +1,36 @@
+packages:
+  - type: github_release
+    repo_owner: atc0005
+    repo_name: send2teams
+    description: Small CLI tool used to submit messages to Microsoft Teams
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 0.4.7")
+        no_asset: true
+      - version_constraint: semver("<= 0.10.3")
+        error_message: |
+          Versions 0.10.3 and earlier include a random string in the asset, making it difficult to predict the filename.
+          To avoid maintaining a registry with numerous overrides, we have decided not to support versions up to and including 0.10.3.
+          Please use a version later than 0.10.3.
+      - version_constraint: "true"
+        asset: send2teams-{{.OS}}-{{.Arch}}.{{.Format}}
+        format: xz
+        files:
+          - name: send2teams
+            src: send2teams-{{.OS}}-{{.Arch}}
+        windows_arm_emulation: false
+        checksum:
+          type: github_release
+          asset: "{{.Asset}}.sha256"
+          algorithm: sha256
+        overrides:
+          - goos: windows
+            asset: send2teams-{{.OS}}-{{.Arch}}.exe.{{.Format}}
+            files:
+              - name: send2teams
+                src: send2teams-{{.OS}}-{{.Arch}}.exe
+        supported_envs:
+          - linux/amd64
+          - linux/386
+          - windows/386
```

Comment:
```suggestion
```

https://github.com/aquaproj/aqua-registry/actions/runs/12222179601/job/34092142647?pr=29681

386 isn't supported.

Reply:
Commands like cmdx do not return any issues related to it. )=